renovate.json

```diff
@@ -16,38 +16,42 @@
 
     "packageRules": [
         {
-            "matchPackagePatterns": ["*"],
+            "matchPackagePatterns": ["npm"],
             "stabilityDays": 3
         },
         {
-            "matchPackagePatterns": ["*"],
-            "matchUpdateTypes": ["patch"],
             "groupName": "all patch updates",
-            "groupSlug": "all-patch"
+            "matchPackagePatterns": ["*"],
+            "matchUpdateTypes": ["patch"]
         },
         {
-            "groupName": "rollup",
-            "groupSlug": "rollup",
-            "matchPackageNames": ["rollup"],
-            "matchPackagePatterns": ["^rollup-", "^@rollup/", "-rollup-"]
+            "groupName": "Bazel",
+            "matchManagers": ["bazel"],
+            "matchUpdateTypes": ["patch", "minor"]
         },
         {
-            "groupName": "Webpack",
-            "groupSlug": "webpack",
-            "matchPackagePatterns": ["^webpack-", "^@webpack-cli/", "-webpack-"]
+            "groupName": "Aspect",
+            "matchSourceUrlPrefixes": ["https://github.com/aspect-build/"],
+            "matchUpdateTypes": ["patch", "minor"],
+            "schedule": null
         },
         {
```

Comment:
From here down is npm specific stuff that probably won't go into non-js related repos.